_drafts/blog-post-1.md

```diff
@@ -0,0 +1,18 @@
+
+Since the outbreak of the novel coronavirus last spring, we have become increasingly dependent on Canvas as a learning tool, and rejecting its widespread use is simply not a viable option. Despite a seeming lack of control over our own data within this system, we shouldn't feel powerless. In fact, now that this information is gradually becoming more transparent, we should feel empowered to make informed decisions based on our new awareness. Part of this awareness is recognizing and questioning the current lack of transparency concerning information practices in higher education. For starters, many students are not aware of the extent by which their data is gathered, not to mention to what end the institution is gathering this information. Should higher education institutions be obligated to inform students of their data collection policies? Following from this, should students be extended the ability to consent to data collection in LMSs? To what degree should they have control over how and what data is analyzed? To what extent should they be allowed to express their personal privacy preferences? Might these considerations undermine the (opaque) goals of learning analytics, which brings us to wonder: are these goals in our best interest, anyway? 
```

Comment:
- _extent by which_: this should be "extent to which"
- An important point in this paragraph that I'd love to see further developed, and perhaps moved more into the foreground, is that students should be asking these questions of their institutions. It's a different question from "Should institutions should be collecting student data?" It points to the need for students to be active in understanding how their data is used across a range of situations and platforms — not simply LMS — and in asking questions of those institutions/organizations/businesses etc. that collect this data.